tsconfig.json

```diff
@@ -18,7 +18,8 @@
     "experimentalDecorators": true,
     "allowUnreachableCode": false,
     "strict": true,
-    "skipLibCheck": true
+    "skipLibCheck": true,
+    "useUnknownInCatchVariables": false
```

Comment:
This is good for now, but would you mind making an issue to remove this (so we don't forget)?

Reply:
yup

Reply:
https://github.com/embroider-build/embroider/issues/952